src/components/NestedForm/NestedFormInput.tsx

```diff
@@ -30,7 +36,7 @@ export class NestedFormInput extends React.PureComponent<data.IFieldInputProps,
         return (
             <div className='form-input-nested'>
                 <div className='form-input-nested-label'>{this.props.field.label}</div>
-                <button type='button' onClick={this.onCreateEntry}>Create</button>
+                {this.renderCreateButton()}
```

Comment:
From our current UI design, the create button is below the all entries, I think it make sense.